fiftyone/utils/labels.py

```
@@ -857,3 +893,10 @@ def _perform_nms(
             del detections[i]
 
     return nms_detections
+
+
+def _get_filepath(sample_or_frame, sample):
+    try:
+        return sample_or_frame.filepath
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)